src/App.tsx

```diff
@@ -10,33 +12,33 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [tabId, setTabId] = useState(tabs[0].id);
+  const [title, setTitle] = useState(tabs[0].title);
+
+  const setAnotherTab = (clickedTab:Tab) => {
+    const anotherTab = tabs.find(tab => tab.id === clickedTab.id);
+
+    if (anotherTab) {
+      (setTabId(anotherTab.id));
+      (setTitle(anotherTab.title));
+    }
+  };
+
+  const onTabSelected = (tab:Tab) => {
```

Comment:
the only thing which this function does it calling another function. So you don't need it. You can move functionality from setAnotherTab to this function directly.